ppml/src/main/cpp/quote_verification.cpp

```diff
@@ -86,21 +86,71 @@ int ecdsa_quote_verification(vector<uint8_t> quote) {
     ret = 0;
     break;
   case SGX_QL_QV_RESULT_CONFIG_NEEDED:
+    printf(
```

Comment:
Move these duplicate print out of switch. Check ret then determine printing success, warning or error.